tests/platformalteration/helper/helper.go

```diff
@@ -114,3 +118,37 @@ func AppendIstioContainerToPod(pod *corev1.Pod, image string) {
 			Command: []string{"/bin/bash", "-c", "sleep INF"},
 		})
 }
+
+// Creates deployment with one pod with one non-UBI based container.
+func DefineDeploymentWithNonUBIContainer() *v1.Deployment {
+	dep := deployment.DefineDeployment(tsparams.TestDeploymentName, tsparams.PlatformAlterationNamespace,
+		tsparams.NotRedHatRelease, tsparams.TnfTargetPodLabels)
+
+	// Workaround as this non-ubi test image needs /bin/sh (busybox) instead of /bin/bash.
+	deployment.RedefineWithContainerSpecs(dep, []corev1.Container{
+		{
+			Name:    "test",
+			Image:   tsparams.NotRedHatRelease,
+			Command: []string{"/bin/sh", "-c", "sleep INF"},
```

Comment:
`sleep` expects an amount of seconds? Probably `while :; do sleep 2073600; done` from here:
 https://stackoverflow.com/questions/2935183/bash-infinite-sleep-infinite-blocking

Reply:
It is not a POSIX/UNIX/macOS: "Although portable POSIX scripts must give sleep a single non-negative integer argument without a suffix, GNU sleep also accepts two or more arguments, unit suffixes, and floating-point numbers in either the current or the C locale". Busybox based Alpine container doesn't support it:
 https://stackoverflow.com/questions/39729241/why-does-the-container-created-with-docker-run-d-alpine-sleep-infinity-goes